core/session.py

```diff
@@ -99,34 +98,37 @@ def save(self, req: BrowseHandler):
                 # We will not issue sessions over http anymore
                 if not (req.isSSLConnection or conf["viur.instance.is_dev_server"]):
                     return False
+
                 # Get the current user id
                 try:
                     # Check for our custom user-api
                     userid = conf["viur.mainApp"].user.getCurrentUser()["key"]
                 except:
                     userid = None
+
                 try:
                     dbSession = db.Entity(db.Key(self.kindName, self.cookieKey))
                     dbSession["data"] = db.fixUnindexableProperties(self.session)
                     dbSession["staticSecurityKey"] = self.staticSecurityKey
                     dbSession["securityKey"] = self.securityKey
-                    dbSession["lastseen"] = time()
-                    # Store the userid inside the sessionobj, so we can kill specific sessions if needed
-                    dbSession["user"] = str(userid) or "guest"
+                    dbSession["lastseen"] = time.time()
+                    dbSession["user"] = str(userid) if userid else None  # to allow filtering for specific users
+                    dbSession["guest"] = userid is None  # to allow filtering guest sessions
```

Comment:
Hmmm. Now we have to fields, while `guest` is always derivable from `user`. Futhermore `killSessionByUser("guest")` doesn't longer work. I don't see any advantage in this second field.

Reply:
Ok, we can turn it back, but then I would like to use `__guest__` as a special meaning identifier.

Reply:
Nevertheless, the previous implementation either wrote the str "None" into the database or the user key's repr (with viur-datastore<1.3.3) or now the user key's url-safe representation.

Reply:
Okay. I think it would be good if this could then be used as a constant. 

```py
GUEST_USER = "__guest__"
```